src/Orleans.CodeGeneration.Build/Orleans.CodeGeneration.Build.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Build;PostBuildPublish">
 
   <PropertyGroup>
-    <PackageId>Microsoft.Orleans.OrleansCodeGenerator.Build</PackageId>
+    <PackageId>Microsoft.Orleans.CodeGeneration.Build</PackageId>
```

Comment:
I'm fine with renaming DLLs, but changing package IDs should be done into its own targeted PR imo. I'm very worried that will end up with dozens of stale package ids in nuget.org just because we like a new name better. I'm not saying we shouldn't do it if it obviously makes sense, but I'm hesitant of jumping the gun quickly so that you don't have to constantly rebase this huge PR that moves every file.
Another similar risk is that we may release beta in a few days by trying to be quick to merge this, and then realize that we need to change it again, and suddenly have several more package IDs that only exist for a beta version

Reply:
I only did what we agreed upon.

Please walk through the changes and try to point out individual packages where you suspect that can "become stale" after preview, because it was not a good rename.

Reply:
Well, that would be 1 of the 2 risks. What about the first one? that we'll have both `Microsoft.Orleans.OrleansCodeGenerator.Build` and `Microsoft.Orleans.CodeGeneration.Build` as listed packages in nuget, and not a completely straightforward way for end users to know which one to choose, especially once we start versioning some nuget packages with different numbers (such as Core.Abstractions). Again, not saying we shouldn't fix a few of them, but not duplicate the number of package IDs just for the sake of refactoring the code.
Splitting this change into its own PR should not be a big deal, so I don't understand the push-back. Unless you are OK with rebasing it until we fully settle on how to move forward, I will definitely not single-handedly merge the PR without full consensus.